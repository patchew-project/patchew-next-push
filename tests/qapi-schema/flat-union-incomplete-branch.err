tests/qapi-schema/flat-union-incomplete-branch.qapi:6: Union 'TestUnion' data missing 'value2' branch
