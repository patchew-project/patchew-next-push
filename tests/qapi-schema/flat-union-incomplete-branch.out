object q_empty
enum QType ['none', 'qnull', 'qnum', 'qstring', 'qdict', 'qlist', 'qbool']
    prefix QTYPE
module flat-union-incomplete-branch.json
enum TestEnum ['value1', 'value2']
object TestTypeA
    member string: str optional=False
object q_obj_TestUnion-base
    member type: TestEnum optional=False
object TestUnion
    base q_obj_TestUnion-base
    tag type
    case value1: TestTypeA
    case value2: q_empty
